nlpsandboxclient/client.py

```diff
@@ -671,3 +672,35 @@ def delete_dataset(host: str, dataset_id: str):
         # Create an instance of the API class
         api_instance = dataset_api.DatasetApi(api_client)
         api_instance.delete_dataset(dataset_id)
+
+
+def evaluate(tool_type: str, host: str, dataset_id: str, annotation_store_id: str):
+    """Evaluate submission annotated notes by comparing them to goldstandards
+    Args:
+        tool_type: Type of annotator
+        host: Data node host IP
+        dataset_id: Dataset Id
+        annotation_store_id: Annotation store Id for the submission  annotations
+    Yields:
+        Performance scores
+
+    Examples:
+        >>> scores = evaluate(host="0.0.0.0/api/v1",
+        >>>                             dataset_id="awesome-dataset",
+        >>>                             annotation_store_id="awesome-store-id"
+        >>>                             tool_type="nlpsandbox:date-annotator")
+    """
+    eval_mapping = {
+        "nlpsandbox:date-annotator": evaluation.DateEvaluation,
+        "nlpsandbox:person-name-annotator": evaluation.PersonNameEvaluation,
+        "nlpsandbox:location-annotator": evaluation.LocationEvaluation,
+        'nlpsandbox:id-annotator': evaluation.IdEvaluation,
+        'nlpsandbox:contact-annotator': evaluation.ContactEvaluation
+    }
+    evaluator = eval_mapping[tool_type]()
+    sys = list_annotations(host, dataset_id, annotation_store_id)
```

Comment:
Ahh, this is definitely a useful function.  `annotation_store_id` here is submission id right?  I thought you wanted to create this function:

```
evaluate_prediction(prediction_json, gold_standard_json, tool_type)
```

Reply:
The function I added will not be used by the workflow. The workflow will still use `evaluate_prediction(prediction_json, gold_standard_json, tool_type)`. I added the function only for the python client. It makes it easier for me to run the evaluation without saving and keeping track of json files.